docs/source/tutorial/trainer.rst

```diff
@@ -0,0 +1,308 @@
+
+7. Start Training
+'''''''''''''''''
+
+Just call :meth:`~chainer.training.Trainer.run` method from
+:class:`~chainer.training.Trainer` object to start training.
+
+.. code-block:: python
+
+    trainer.run()
+
+::
+
+    epoch       main/loss   main/accuracy  validation/main/loss  validation/main/accuracy  elapsed_time
+    1           1.53241     0.638409       0.74935               0.835839                  4.93409
+    2           0.578334    0.858059       0.444722              0.882812                  7.72883
+    3           0.418569    0.886844       0.364943              0.899229                  10.4229
+    4           0.362342    0.899089       0.327569              0.905558                  13.148
+    5           0.331067    0.906517       0.304399              0.911788                  15.846
+    6           0.309019    0.911964       0.288295              0.917722                  18.5395
+    7           0.292312    0.916128       0.272073              0.921776                  21.2173
+    8           0.278291    0.92059        0.261351              0.923457                  23.9211
+    9           0.266266    0.923541       0.253195              0.927314                  26.6612
+    10          0.255489    0.926739       0.242415              0.929094                  29.466
+
+Let's see the plot of loss progress saved in the ``mnist_result`` directory.
+
+.. image:: ../../image/trainer/mnist_loss.png
+
+How about the accuracy?
+
+.. image:: ../../image/trainer/mnist_accuracy.png
+
+Furthermore, let's visualize the computaional graph saved with :meth:`~chainer.training.extensions.dump_graph` using Graphviz.
+
+::
+
+    % dot -Tpng mnist_result/cg.dot -o mnist_result/cg.png
+
+.. image:: ../../image/trainer/mnist_graph.png
+
+From the top to the bottom, you can see the data flow in the computational graph. It basically shows how data and parameters are passed to the :class:`~chainer.Function`\ s.
+
+8. Evaluate a pre-trained model
+'''''''''''''''''''''''''''''''
+
+Evaluation using the snapshot of a model is as easy as what explained in the :doc:`train_loop`.
+
+.. code-block:: python
+
+    import matplotlib.pyplot as plt
+
+    model = MLP()
+    serializers.load_npz('mnist_result/model_epoch-10', model)
+
+    # Show the output
+    x, t = test[0]
+    plt.imshow(x.reshape(28, 28), cmap='gray')
+    plt.show()
+    print('label:', t)
+
+    y = model(x)
+
+    print('predicted_label:', y.argmax(axis=1)[0])
+
+.. image:: ../../image/trainer/mnist_output.png
+
+::
+
+    label: 7
+    predicted_label: 7
+
+The prediction looks correct. Yatta!
```

Comment:
English speakers won't understand "Yatta" and the top google response for it is [this video](https://www.youtube.com/watch?v=rW6M8D41ZWU). That ok for you?